crates/rover-client/src/operations/subgraph/check_workflow/runner.rs

```diff
@@ -158,6 +165,17 @@ fn get_check_response_from_data(
                 proposals_target_url = task.target_url;
                 proposals_result = Some(typed_task);
             }
+            CustomCheckTask(typed_task) => {
+                custom_status = Some(task.status);
+                custom_target_url = task.target_url;
+                if let Some(result) = typed_task.result {
+                    custom_result = Some(result)
+                } else {
+                    return Err(RoverClientError::MalformedResponse {
+                        null_field: "graph.checkWorkflow....on CustomCheckTask.result".to_string(),
+                    });
+                }
```

Comment:
Could we simplify this too and use something like `.or_else` or a similar variant?

Reply:
I don't think this is even necessary. Going to fully remove it.